detox/test/src/Screens/MatchersScreen.js

```diff
@@ -21,11 +21,11 @@ export default class MatchersScreen extends Component {
       <View style={{flex: 1, paddingTop: 20, justifyContent: 'center', alignItems: 'center'}}>
 
         <TouchableOpacity onPress={this.onButtonPress.bind(this, 'Label Working')}>
-          <Text style={{color: 'blue', marginBottom: 20}}>Label</Text>
+          <Text style={{color: 'blue', marginBottom: 20}} accessibilityLabel={'Label'}>Label</Text>
         </TouchableOpacity>
 
         <TouchableOpacity onPress={this.onButtonPress.bind(this, 'ID Working')}>
-          <Text testID='UniqueId345' style={{color: 'blue', marginBottom: 20}}>ID</Text>
+          <Text testID='UniqueId345' style={{color: 'blue', marginBottom: 20}} accessibilityLabel={'ID'}>ID</Text>
```

Comment:
@simonracz explain, please, why `testID` and `accessibilityLabel` have different labels? I'm using [`react-native-testid`](https://github.com/isnifer/react-native-testid) at Tipsi. And we have both parameters the same

Reply:
AccessibilityLabel is user visible. TestId is not.

Reply:
@isnifer Bad idea.
https://github.com/isnifer/react-native-testid/issues/1